setup.py

```diff
@@ -27,14 +27,14 @@ def run(self):
         )
         from django.core.management import call_command
         import django
-        if django.VERSION[:2] >= (1, 7):
+        if django.VERSION[:2] >= (1, 8):
```

Comment:
I'm not sure I totally follow. Are you saying there is a bug in the patch, or in the tests?